session_3/README.md

```diff
@@ -0,0 +1,38 @@
+# Session 3 - Controller, Devise

```

Comment:
God, that's horrible!

We are not shouting down commands for people. Please explain what you are asking them to do.

Reply:
@abhishekkumar2718 The documentation is not complete yet, with the explanation for each step, this is just an initial version